cpp/src/arrow/adapters/arrow-rados-cls/docs/benchmarking.md

```diff
@@ -0,0 +1,68 @@
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+# Setting up and benchmarking SkyhookDM
+
+1. Download the required scripts and make them executable.
+
+```bash
+wget https://raw.githubusercontent.com/uccross/skyhookdm-arrow/arrow-master/cpp/src/arrow/adapters/arrow-rados-cls/scripts/deploy_ceph.sh && chmod +x deploy_ceph.sh
+wget https://raw.githubusercontent.com/uccross/skyhookdm-arrow/arrow-master/cpp/src/arrow/adapters/arrow-rados-cls/scripts/deploy_skyhook.sh && chmod +x deploy_skyhook.sh
+wget https://raw.githubusercontent.com/uccross/skyhookdm-arrow/arrow-master/cpp/src/arrow/adapters/arrow-rados-cls/scripts/deploy_data.sh && chmod +x deploy_data.sh
+```
+
+2. Execute deploy_ceph script to deploy a Ceph cluster on a set of nodes and to mount CephFS on the client/admin node. On the client node, execute:
+
+```bash
+./deploy_ceph.sh mon1,mon2,mon3 osd1,osd2,osd3 mds1 mgr1
+```
+where mon1, mon2, osd1, etc. are the internal hostnames of the nodes.
+
+3. Build and install the SkyhookDM CLS plugins and PyArrow (with Rados Parquet extensions):
+
+```bash
+./deploy_skyhook.sh osd1,osd2,osd3 arrow-master true true
+```
+This will build the CLS plugins as shared libraries and deploy them to the OSD nodes.
+
+4. Pull datasets from the [datasets repository](https://github.com/JayjeetAtGithub/datasets):
```

Comment:
"Download a sample dataset from [this](https://github.com/jayjeetc/datasets) repository:"

Reply:
Done.